siphon/tests/test_catalog.py

```diff
@@ -221,6 +221,46 @@ def test_datasets_time_range():
                       'NAM_CONUS_20km_noaaport_20150529_0000.grib1']
 
 
+@recorder.use_cassette('top_level_20km_rap_catalog')
+def test_datasets_time_range_regex():
+    """Test getting datasets by time range using filenames, with manual regex"""
```

Comment:
Bot's likely to complain about the lack of the period at the end of the docstring.

Reply:
Fixed in 5fa3b9b